src/main/java/pl/antonina/tasks/child/ChildController.java

```diff
@@ -15,27 +17,27 @@ public ChildController(ChildService childService) {
     }
 
     @GetMapping
-    public List<ChildView> getChildByParentId(@RequestParam long parentId) {
-        return childService.getChildrenByParentId(parentId);
+    public List<ChildView> getChildrenByParent(@ApiIgnore Principal principal) {
```

Comment:
Bardzo fajne zmiany nazw zmiennych, `childId` jest dużo lepsze od `id`. Tu też by można dać coś więcej mówiącego, np. `parentPrincipal` (i wszędzie niżej).

Reply:
Dobry pomysł, bo czasami principal - to Child... żeby później się nie mylić. Poprawię.